src/components/GameView/CannotCounterDialog.vue

```diff
@@ -11,10 +11,11 @@
 			<v-card-title>Cannot Counter</v-card-title>
 			<v-card-text>
 				<span v-if="!opponentLastTwo">Your opponent has played the {{ oneOff.name }} as a one-off
-					<span v-if="target"> targetting your {{ target.name }}</span>
+					<span v-if="target"> targeting your {{ target.name }}</span>
 				</span>
 				<span v-else>
-					Your opponent has played {{ opponentLastTwo.name }} to Counter<span v-if="playerLastTwo"> your {{ playerLastTwo.name }}</span>.
+					Your opponent has played {{ opponentLastTwo.name }} to
+					Counter<span v-if="playerLastTwo"> your {{ playerLastTwo.name }}</span>.
```

Comment:
Agh, I wrote out a response but guess I forgot to post it and now it's lost :( .  Time to rewrite it!

----

In short, I was unable.  Since HTML collapses new lines to just a single whitespace, the `.` has to be on the same line as `Counter`.  I tried playing with the CSS in the `span v-else` part of things with [`white-space:nowrap;`](https://stackoverflow.com/questions/7300760/prevent-line-break-of-span-element), but couldn't get it working for some reason.  Removing the whitespace at the beginning of the conditional is easy (change `_your_` to just `your_`).  Having there be no whitespace before the `.` without the conditional is more complicated.

Reply:
No worries, forget it